sopel/modules/find.py

```diff
@@ -50,10 +50,11 @@ def collectlines(bot, trigger):
     # Update in-memory list of the user's lines in the channel
     line_list = bot.memory['find_lines'][trigger.sender][trigger.nick]
     line = trigger.group()
-    if line.startswith("s/"):  # Don't remember substitutions
+    if line.startswith('s/') or line.startswith('s|'):
+        # Don't remember substitutions
         return
     # store messages in reverse order (most recent first)
-    elif line.startswith("\x01ACTION"):  # For /me messages
+    elif line.startswith('\x01ACTION'):  # For /me messages
```

Comment:
Unrelated style change :|
Exi I thought you were above that 😹

Reply:
Probably a minor change I didn't think through that much.